staging/kubefed/values.yaml

```diff
@@ -33,10 +33,19 @@ controllermanager:
     adoptResources:
   ## Value of feature gates item should be either `Enabled` or `Disabled`
   featureGates:
-    PushReconciler:
-    SchedulerPreferences:
-    CrossClusterServiceDiscovery:
-    FederatedIngress:
+    featureGates:
```

Comment:
This is repetition of featureGates property above. I don't think we need this change at all and shouldn't set the featureGates values, leave blank so this is easier to upstream.

Reply:
Sure, I left these for testing in our CI. Locally i could not test because these values needed to be filled in. I will attempt to see if we can test.

Reply:
Testing in CI can be done by specifying values files for various CI configurations - no need for this to be in values to test in CI.